Applications/Games/League of Legends/application.json

```diff
@@ -2,4 +2,5 @@
     "name"        : "League of Legends",
     "id"          : "league_of_legends",
     "description" : "League of Legends is a fast-paced, competitive online game that blends the speed and intensity of an RTS with RPG elements. Two teams of powerful champions, each with a unique design and playstyle, battle head-to-head across multiple battlefields and game modes. With an ever-expanding roster of champions, frequent updates and a thriving tournament scene, League of Legends offers endless replayability for players of every skill level."
+    "testingOperatingSystems": [LINUX],
```

Comment:
This JSON is not valid. You need to add quotes mark around LINUX

Reply:
Pushed https://github.com/PhoenicisOrg/scripts/pull/782/commits/6642c320a17a0b7152db3ff908018146e8be10ef

requesting verification